ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -143,6 +144,41 @@ objects:
           oc create -f /tmp/cluster/insights-live.yaml || true
         fi
 
+        # create fips enable helpers
+        function enable_fips_master() {
+            cat > /tmp/master-fips-mc.yaml <<'EOF'
+          apiVersion: machineconfiguration.openshift.io/v1
+          kind: MachineConfig
+          metadata:
+            labels:
+              machineconfiguration.openshift.io/role: master
+            name: 99-fips-master
+          spec:
+            fips: true
+          EOF
+            oc create -f /tmp/master-fips-mc.yaml
+
+            for i in $(seq 0 10); do oc wait machineconfigpool/master --for=condition=Updating --timeout=5m && break; done
+            for i in $(seq 0 10); do oc wait machineconfigpool/master --for=condition=Updated --timeout=5m && break; sleep 30; done
+        }
+
+        function enable_fips_worker() {
```

Comment:
why do you have two functions?  You need to do this for all machine pools, so you need to be getting all the pools and looping over them.

Reply:
Should just be one function, set all pools at once and loop over all of them to wait.

Reply:
Changed it to a single function that waits for all pools